Dockerfile

```diff
@@ -3,13 +3,13 @@ FROM python:3.11-slim-bookworm
 
 # Set working directory and copy required files
 WORKDIR /app
-COPY ./requirements.txt /app/requirements.txt
+COPY ./src/requirements.txt /app/requirements.txt
```

Comment:
Why is the COPY necessary? Why not just have the requirements txt at the root level? or move the DockerFile to the src level?